packages/core/less/treegrid-table.less

```diff
@@ -0,0 +1,87 @@
+.chevron-position(@shift-left) {
+  left: -(@table-cell-padding*2.2 + @shift-left);
+}
+
+@show-less-icon: '\f105';
+@show-more-icon: '\f107';
```

Comment:
Might make these more specific so we don't accidentally overwrite any application settings.